prebuild/macOS/.gitignore

```diff
@@ -0,0 +1 @@
+macdylibbundler/
```

Comment:
This was vestigial, should be able to remove this since it uses `macpack` now

Reply:
Removed.